### ISSUE TITLE
How can I make this model converges in the custom data?

### ISSUE THREAD

**Issue by hse801** [eriklindernoren/PyTorch-YOLOv3#672]
I'm training with my custom data, and when I train with exact same data in ultralytics repo it converges to about 0.7 mAP, but with this repo mAP remains around 0.1. But some adaptation I want to make is much more easier in this repository so I want to fix and use this repo if it's possible. 
Anyone knows the reason? I noticed the same issue below, but there was no clear answer. How should I fix the model?

**Comment by Flova** [comment]
Thanks for your issue. I have a few questions. 
* What model did you use?  
* What's the size of your dataset? 
* What learning rate and batch size did you use? 
* What's the current HEAD of your git? 
* Could you provide some tensorboard logs?

Regarding the convergence. Newer versions of the repo adapted the loss function with features of the Ultralytics one which is originally also adapted from this repo. 

But while doing inference I noticed some weird behavior. If load darknet weight (the official coco ones) and save them again they are identical to the bit. But if I run this repo and darkent on these weights I get slightly different results. Not much, but also not identical. So I disabled nms to see if the other nms approach resulted in this. And no some boxes where slightly shifted with to way that this nms could merge or select them in a way that results in the darknet boxes. So I thought of a different padding approach in the convolutions, but this made the problem worse. Same with different upscaling techniques. It's weird because is not like the boxes **all** have a offset due to a wrong grid or something similar.

I would be very lucky if somebody had an idea what else could result in this. Sorry for the off topic but I think this might be related. On the other hand it seems like your difference is much worse than what I have observed.

**Comment by Flova** [comment]
Did you use any pretrained weights?
